odoo/addons/base/models/res_partner.py

```diff
@@ -62,7 +62,9 @@ def _view_get_address(self, arch):
                         self.env['ir.ui.view'].postprocess_and_fields(sub_arch, model=self._name)
                     except ValueError:
                         return arch
-                address_node.getparent().replace(address_node, sub_arch)
+                new_address_node = sub_arch.find('.//div[@class="o_address_format"]')
```

Comment:
If we can't find it, should be use the whole "sub arch" like before ? To be more stable compliant.